### PR TITLE
Complete SDG Index collection

### LINKED ISSUE #26: Collect historic SDG Index data
The SDG index has [published reports back to 2015](https://www.sdgindex.org/reports/). They are not in uniform formats however.

- [ ] 2018
- [ ] 2017
- [ ] 2016
- [ ] 2015 

**To Do**
- [ ] Download
- [ ] Parse and transform
- [ ] Load

### LINKED ISSUE #27: Collect 2020 SDG Index data
This shouldn't be too hard as it seems to follow the same format as 2019.

Find the report and data [here](https://www.sdgindex.org/reports/sustainable-development-report-2020/).

**To Do**
- [ ] Collect
- [ ] Parse and transform
- [ ] Load

### LINKED ISSUE #29: Create mappings for SDG Index indicator names
The SDG index datasets contain codebooks which have names for all of the abbreviated column names of the index. These would be useful to have in a json/dict format for easily importing and using for plots etc.

### PR DESCRIPTION
**Pull request by georgerichardson**
Add ability to download, parse and load historic and 2020 SDG Index data. Also add functionality to load 2019 and 2020 SDG Index codebooks and raw data.

closes #27 
closes #26 
closes #29

### CODE REVIEW

File: sdg_mapping/sdg_index/sdg_index_utils.py [hunk 1]
```diff
@@ -59,16 +66,66 @@ def parse_2019_sdg_indicator(dataset):
...
+        year (int): Year of the index. 
+
+    Returns:
+        df (pd.DataFrame): Clean data.
+    """
+    df = _map_dashboard_symbols(df)
+    df.columns = [c.lower().replace(" ", "_") for c in df.columns]
+    if year == 2017:
+        old = "average_score_on_sdg"
+        new = "goal_"
```
Comment (dede95) [inline sdg_mapping/sdg_index/sdg_index_utils.py:102]: Is the underscore in `goal_` needed?

File: sdg_mapping/sdg_index/sdg_index_utils.py [hunk 2]
```diff
@@ -59,16 +66,66 @@ def parse_2019_sdg_indicator(dataset):
...
+        df.columns = [c.replace(old, new) + "_score" for c in df.columns]
```
Comment (dede95) [inline sdg_mapping/sdg_index/sdg_index_utils.py:103]: Is the underscore in `goal_` needed?

Comment (georgerichardson) [reply sdg_mapping/sdg_index/sdg_index_utils.py:103]: Yeah. The originals are like `average_score_on_sdg11` and to be consistent with other years the result needs to be `goal_11_score`

Comment (dede95) [reply sdg_mapping/sdg_index/sdg_index_utils.py:103]: Oh right, that makes sense.

**Review by dede95 — APPROVED** [review]

**Pull request merged**